parsigs/parse_sig_api.py

```diff
@@ -55,6 +55,7 @@ class StructuredSig:
 
 
 dose_instructions = ['take', 'inhale', 'instill', 'apply', 'spray', 'swallow']
+dose_forms = ["tablet", "puff", "pill", "capsule"]
 number_words = ["one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten"]
```

Comment:
@oren-ron This fix is not enough as there can be more forms then what you have inserted (Injection, ampule and more) 

We need to think of a more general way mitigating the "a" use case

Reply:
Do you want me to try and come up with a full list of possibilities or do we need to update the model so it will recognize it automatically? If it's related to the model itself I'd love to help but I need to learn how to do it